assets/js/sort-adr.js

```diff
@@ -0,0 +1,62 @@
+function onToggle(elementId) {
+  document.getElementById(elementId).classList.toggle('hidden');
+}
+
+function sortBy(sortType) {
+  // Select all elements with class 'adr-links' and convert NodeList to array
+  let adrEntries = Array.from(
+    document.getElementById('adr-links').querySelectorAll('a'),
```

Comment:
We can avoid redundant calls to `document.getElementById('adr-links')`. We can store it in a variable and use it in other places.

`const adrContainer = document.getElementById('adr-links');`

Reply:
Thank you @zeshanziya for the review. I'll make necessary changes in upcoming commits.